### PR TITLE
use less data to run agg spill ut

### PR DESCRIPTION
**Pull request by windtalker**
### What problem does this PR solve?

Issue Number: ref #6233

Problem Summary:

### What is changed and how it works?
Use less data to run agg spil ut
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Flash/tests/gtest_spill_aggregation.cpp [hunk 1]
```diff
@@ -116,7 +116,7 @@ try
     /// test spill with small max_block_size
     /// the avg rows in one bucket is ~10240/256 = 400, so set the small_max_block_size to 300
```
Comment (Lloyd-Pottiger) [inline dbms/src/Flash/tests/gtest_spill_aggregation.cpp:117]:
```suggestion
    /// the avg rows in one bucket is ~10240/256 = 400, so set the small_max_block_size to 100
```

**Review by Lloyd-Pottiger — APPROVED** [review]
Others LGTM

**Review by yibin87 — APPROVED** [review]
LGTM

**Pull request merged**